src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -120,6 +122,7 @@ public boolean equals(Object other) {
      * corresponding field value of the student.
      */
     public static class EditStudentDescriptor {
+        private UUID uuid;
```

Comment:
I don't think there is a need for `EditStudentDescriptor` to have a `uuid` field.

Reply:
Resolved by removing UUID field